std_msgs/Header header
uint32 height           # image height in original resolution
uint32 width            # image width in original resolution
uint32 target_x         # x pixel of center of target in image - in original image resolution
uint32 target_y         # y pixel of center of target in image - in original image resolution
uint32 target_size      # size of object found in image, in num pixels of image downsized to 320 x 240